src/js/components/homepage/hero/Hero.jsx

```diff
@@ -65,7 +65,7 @@ export default class Hero extends React.Component {
                         {tooltip}
                         <hr className="homepage-hero__divider" />
                         <div className="homepage-hero__description">
-                            <strong className="homepage-hero__description homepage-hero__description_weight_bold">Curious to see how this money was spent?</strong> We hope so &mdash; we&apos;ve opened the conversation around federal spending and provide the tools to help you navigate the budget from top to bottom.
+                            <strong className="homepage-hero__description homepage-hero__description_weight_bold">As the authoratitative resource for federal funding,</strong> USA Spending increases government transparency by making the data accessible, searchable, and reliable. See how the money was spent with tools to help you navigate spending from top to bottom.
```

Comment:
- "authoritative" misspelled

The mock shows:
- "resource for federal **spending**" instead of funding
- "this data" instead of "the data"